sdk/app_cpu1/common/drv/gp3io_mux.h

```diff
@@ -12,10 +12,6 @@
 #define GP3IO_MUX_2_BASE_ADDR (XPAR_HIER_GPIO_1_AMDC_GP3IO_MUX_0_S00_AXI_BASEADDR)
 #define GP3IO_MUX_3_BASE_ADDR (XPAR_HIER_GPIO_2_AMDC_GP3IO_MUX_0_S00_AXI_BASEADDR)
 #define GP3IO_MUX_4_BASE_ADDR (XPAR_HIER_GPIO_3_AMDC_GP3IO_MUX_0_S00_AXI_BASEADDR)
-#define GP3IO_MUX_5_BASE_ADDR (XPAR_HIER_GPIO_4_AMDC_GP3IO_MUX_0_S00_AXI_BASEADDR)
-#define GP3IO_MUX_6_BASE_ADDR (XPAR_HIER_GPIO_5_AMDC_GP3IO_MUX_0_S00_AXI_BASEADDR)
-#define GP3IO_MUX_7_BASE_ADDR (XPAR_HIER_GPIO_6_AMDC_GP3IO_MUX_0_S00_AXI_BASEADDR)
-#define GP3IO_MUX_8_BASE_ADDR (XPAR_HIER_GPIO_7_AMDC_GP3IO_MUX_0_S00_AXI_BASEADDR)
```

Comment:
Hmm, can you help me understand this?

Reply:
Is it true that each `GP3IO_MUX` matches to 1 hardware GPIO port? so, there are 4 total.

Then....why did the code have 8 total before? 🤔

Reply:
You are correct. Because whoever wrote this code originally (me) didn't know what he was doing 😂

Reply:
Ha, ok, I see. Was there extra FPGA IP blocks that you also removed in doing this? Otherwise, it wouldn't have compiled?

Reply:
Hmmm... no, I didn't remove anything from the FPGA. And somehow did compile before. Maybe because the `GP3IO_MUX_5..8_BASE_ADDR` definitions are never used. So it doesn't matter if the values don't exist in `xparameters.h`